DUI3-DX/DUI3/Speckle.Connectors.DUI/Bridge/SyncToUIThread.cs

```diff
@@ -19,8 +20,15 @@ public Task<T> RunOnThread<T>(Func<T> func)
 
     _bridge.RunOnMainThread(() =>
     {
-      T result = func.Invoke();
-      tcs.SetResult(result);
+      try
+      {
+        T result = func.Invoke();
+        tcs.SetResult(result);
+      }
+      catch (Exception ex)
```

Comment:
Blocker warning/ci error here, but imho valid place to overrule it, as it's doing the right thing (as far as I understood it!)

Reply:
seems legit